gcs-fetcher/cmd/gcs-fetcher/main.go

```diff
@@ -89,9 +112,11 @@ func main() {
 		Backoff:     *backoff,
 		SourceType:  *sourceType,
 		Verbose:     *verbose,
+		Stdout:      stdout,
+		Stderr:      stderr,
 	}
 	if err := gcs.Fetch(ctx); err != nil {
-		log.Fatal(err)
+		logFatalf(stderr, err.Error())
```

Comment:
How about `logFatalf(stderr, "Fetch: %v", err)` ?

Reply:
Sure, but you just proved my point below. :)